openedx/features/content_type_gating/partitions.py

```diff
@@ -17,9 +17,11 @@
 
 from course_modes.models import CourseMode
 from lms.djangoapps.commerce.utils import EcommerceService
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.lib.mobile_utils import is_request_from_mobile_app
 from openedx.features.content_type_gating.helpers import CONTENT_GATING_PARTITION_ID, FULL_ACCESS, LIMITED_ACCESS
 from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from openedx.features.discounts.applicability import discount_percentage, can_receive_discount
```

Comment:
Can you test this similar to how we are testing the course home? (openedx/features/course_experience/tests/views/test_course_home.py--test_first_purchase_offer_banner_display() ) Basically, this test should not care about the implementation of the can_receive_discount() function.